xml/art_virtualization-best-practices.xml

```diff
@@ -560,11 +559,11 @@ always madvise [never]</screen>
       KSM is a good technique to over-commit host memory when running multiple
       instances of the same application or &vmguest;. When applications and
       &vmguest; are heterogeneous and do not share any common data, it is
-      preferable to disable KSM. In order to do that, run:
+      preferable to disable KSM. In order to do that, run
```

Comment:
Removing the colon makes it inconsistent with the other screens. It was suggested because of the 'or' in line 566, but I would rather leave the colon in place and change the 'or'.
```suggestion
      preferable to disable KSM. In order to do that, run:
```